cartography/intel/aws/iam.py

```diff
@@ -224,6 +224,18 @@ def load_groups(neo4j_session, groups, current_aws_account_id, aws_update_tag):
         )
 
 
+@timeit
```

Comment:
You can remove the `@timeit` here, I don't think this function is slow enough to have its own timer